### PR TITLE
Put extensions in list

### PR DESCRIPTION
**Pull request by dway123**
I'm not sure these extensions can be used outside of a list.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The following web application declares in its manifest that it can handle CSV an
           "action": "/open-svg",
```
Comment (chasephillips) [inline explainer.md:45]: Can you update line 35 to remove the spaces between []?

**Review by chasephillips — APPROVED** [review]
Ah, makes sense.

LGTM w/ a change

**Comment by mkruisselbrink** [comment]
For consistency with the File System Access API you might want to allow both with and without a array? In fact in https://github.com/WICG/file-system-access/issues/197 we changed FSA to be consistent with what is in the file handling explainer, and with that already shipped I think it would be better to keep them consistent?

**Comment by dway123** [comment]
Ah, I'm not sure whether we allow both with/without an array... lemme check then.

**Comment by dway123** [comment]
Hmm, it looks like Chrome already allows both with and without an array (at least testing via https://morning-bubbly-chauffeur.glitch.me/ on MacOS / Chrome 89.0.4389.82), so this change is unnecessary. I just wasn't sure how things worked or what the limitations were here oops. Thank you for pointing this out, Marijn!

**Comment by dway123** [comment]
I'll cancel/close this PR

**Pull request closed without merging**